index.js

```diff
@@ -9,6 +9,7 @@ import fs from 'fs';
 
 const HOST = process.env.HOST || "0.0.0.0";
 const PORT = parseInt(process.env.PORT) || 3000;
+const FORKS = parseInt(process.env.FORKS) || 0;
```

Comment:
wouldn't it be clearer for the default to be -1 - I know it doesn't matter but it improves readability a bit doesn't it?